src/main/java/com/hillel/items_exchange/config/SecurityConfig.java

```diff
@@ -27,6 +28,7 @@
     public static final String HAS_ROLE_USER = "hasRole('ROLE_USER')";
     private final JwtTokenProvider jwtTokenProvider;
     private final JwtAuthenticationEntryPoint jwtAuthenticationEntryPoint;
+    private final InvalidatedTokensHolder invalidatedTokensHolder;
```

Comment:
Unused bean.

Reply:
I've tried to extract it but @RequiredArgsConstructor said that he needs it.